_posts/en/posts/2016-02-18-release-0-12-0.md

```diff
@@ -0,0 +1,109 @@
+
+_Credits: Pieter Wuille, Greg Maxwell, and Cory Fields_
+
+## Ability to Limit Upload Traffic
+
+Node upload traffic can be burdensome for some users, so the ability to put limitations on such traffic is a much-needed improvement. Node users now have the ability to set soft limits on how much data they upload and serve to their peers. Users can set a parameter that specifies how much data the node should target to be served daily. It will try to stay below the limit, rather than hit it, and if it hits the limit it will only serve blocks requested within the last week.
+
+_Credits: Jonas Schnelli_
+
+Crash Prevention via Memory Pool Limits
+
+Older versions of Bitcoin Software had no limit on the number of transactions they would allow into their memory pool. Even though nodes will only accept transactions that have a certain specified minimum relay fee, at times the number of transactions that meet those requirements will get arbitrarily large and cause nodes with relatively low RAM to crash. Particularly concerning is that attackers can take advantage of this system by flooding the network with transactions in order to crash a subset of nodes.
+
+With this update, nodes now have default limitations on the size of their memory pools and the operator can configure this to the amount of memory they want to dedicate to the mempool. When the memory limit is reached, new transactions can still be accepted, while transactions with the lowest fees will be dropped from the mempool. This new memory limitation ensures that unexpected crashes will not happen due to the number of cached transactions getting out of hand.
+
+_Credits: Matt Corallo and Suhas Daftuar_
+
+## Option to Send Transactions That Can Be Fee-Boosted
+
+Transactions often get stuck if they are have fees that are too low. This can cause problems because unspent transactions outputs (UTXOs) that were used in those transactions can be hard to spend, potentially freezing funds. Appropriate transaction fees are hard to calculate because they are highly dependent on the volume of transactions and their fees at any given time. Thus, one either typically undershoots and gets many stuck transactions or often overshoots, resulting in a massive overpayment and a steady loss of funds.
+
+A new feature called Opt-in Replace-by-Fee gives transaction senders the option to configure their transactions to be able to be replaced later by other transactions that specify larger fees. Senders can start with a low fee and see if their transaction gets picked up, then increase their fee until it gets accepted. This allows senders to both minimize the fees they pay and maximize the chance that their transactions will be included in a block.
+
+_Credits: Peter Todd and Suhas Daftuar_
+
+## Improved Rules for Transaction Relaying
```

Comment:
I'm not sure what this is referring to exactly.  The fact that free transactions don't always get relayed is a byproduct of mempool limiting, not exactly a feature.  The limitations on chains of unconfirmed transactions is a feature, but that doesn't have anything to do with fees.